components/dashboard/ExpenseReporter/index.tsx

```diff
@@ -0,0 +1,46 @@
+import { gql, useQuery } from '@apollo/client'
+import React from 'react'
+import { Bar, BarChart, CartesianGrid, Legend, ResponsiveContainer, Tooltip, XAxis, YAxis } from 'recharts'
+
+import DateHelper from '../../../lib/date-helper'
+
+function ExpenseReporter() {
+  const today = new Date()
```

Comment:
if we are using moment then use moment everywhere instead of Date, to be consistent. I will suggest export a method today() from the dateHelper so that the implementation can be modified as and when needed